src/lib/server/Config.h

```diff
@@ -203,7 +203,7 @@ class Config
   virtual ~Config();
 
 #ifdef TEST_ENV
-  Config() : m_inputFilter(NULL)
+  Config() : m_inputFilter(nullptr), m_mouseSpeed(1.0)
```

Comment:
Unnecessary, already using default init.

```suggestion
  Config() : m_inputFilter(nullptr)
```